site/pages/create-consequence-stops.page.tsx

```diff
@@ -0,0 +1,318 @@
+import Table from "../components/form/Table";
+import TextInput from "../components/form/TextInput";
+import TimeSelector from "../components/form/TimeSelector";
+import { BaseLayout } from "../components/layout/Layout";
+import {
+    CONSEQUENCE_TYPES,
+    COOKIES_CONSEQUENCE_INFO,
+    COOKIES_CONSEQUENCE_TYPE_INFO,
+    DISRUPTION_SEVERITIES,
+    VEHICLE_MODES,
+    COOKIES_CONSEQUENCE_STOPS_ERRORS,
+} from "../constants";
+import { CreateConsequenceProps, PageState } from "../interfaces";
+import { createConsequenceStopsSchema, stopsImpactedSchema } from "../schemas/create-consequence-stops.schema";
+import { typeOfConsequenceSchema } from "../schemas/type-of-consequence.schema";
+import { flattenZodErrors, getDisplayByValue, getPageStateFromCookies } from "../utils";
+import { getStateUpdater } from "../utils/formUtils";
+
+const title = "Create Consequence Stops";
+const description = "Create Consequence Stops page for the Create Transport Disruptions Service";
+
+export interface ConsequenceStopsPageInputs extends Partial<z.infer<typeof createConsequenceStopsSchema>> {}
+export interface Stop extends z.infer<typeof stopsImpactedSchema> {}
+
+const CreateConsequenceStops = ({
+    inputs,
+    previousConsequenceInformation,
+}: CreateConsequenceProps<ConsequenceStopsPageInputs>): ReactElement => {
+    const [pageState, setPageState] = useState<PageState<Partial<ConsequenceStopsPageInputs>>>(inputs);
+    const stateUpdater = getStateUpdater(setPageState, pageState);
+    const [selected, setSelected] = useState<SingleValue<Stop>>(null);
+
+    const getOptionLabel = (stop: Stop) => {
+        if (stop.commonName && stop.indicator && stop.atcoCode) {
+            return `${stop.commonName} ${stop.indicator} ${stop.atcoCode}`;
+        } else if (stop.commonName && stop.atcoCode) {
+            return `${stop.commonName} ${stop.atcoCode}`;
+        } else {
+            return "";
+        }
+    };
+
+    const handleChange = (value: SingleValue<Stop>) => {
+        if (
+            !pageState.inputs.stopsImpacted ||
+            pageState.inputs.stopsImpacted.filter((data) => data.id === value?.id).length === 0
+        ) {
+            addStop(value);
+        }
+        setSelected(null);
+    };
+
+    const production = process.env.NODE_ENV === "production";
+
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    const loadOptions = async (inputValue: string, _callback: (options: Stop[]) => void) => {
+        if (inputValue.length >= 3) {
+            const searchApiUrl = !production
+                ? "https://api.test.ref-data.dft-create-data.com/v1/stops?adminAreaCode=099"
+                : `https://api.${process.env.NODE_ENV}.ref-data.dft-create-data.com/v1/stops?adminAreaCode=099`;
+            const limit = 10;
+            const queryAdder = !searchApiUrl.includes("?") ? "?" : "&";
+            const fetchURL = `${searchApiUrl}${queryAdder}search=${inputValue}&limit=${limit}`;
```

Comment:
`limit` isn't going to be doing anything here, I think we can probably simplify this to something like:
```ts
const searchApiUrl = `${API_BASE_URL}/stops?adminAreaCodes=${ADMIN_AREA_CODE}&search=${inputValue}`
```
we shouldn't need the `queryAdder`

Reply:
done